guides/create-a-component.mdx

```diff
@@ -0,0 +1,311 @@
+---
+title: "Create a Component"
+
+---
+An Architect component is comprised of one or more services that live and die together. The services
+are tied together via the `architect.yml` file,  and one Architect component may
+depend on another by specifying it in the [`dependencies`](../components/dependencies) block. This guide uses the
+[React starter project](https://github.com/architect-templates/react) to help you learn about defining
+an Architect component.
+
+### Prerequisites
+
+The following prerequisites are required for a smooth introduction to Architect:
+
+- Some familiarity with running applications using Docker (or any other similar
+  container engine)
+- [Docker](https://docs.docker.com/get-docker/) installed and running
+- [Architect CLI](https://github.com/architect-team/architect-cli) installed
+
+You don't need to know how to use Kubernetes or how to deploy an application
+into a cloud environment - Architect handles that for you.
+
+## Get Started
+
+A component includes all the services and dependencies that comprise your application.
+Creating a new component that can be run in the Architect cloud as well as any external
+clusters you add involves the following steps:
+
+- Writing an `architect.yml` file
+- Running and testing the component locally with `architect dev`
+- Using the `architect register` command to register this component with your
+  Architect account
```

Comment:
@DevMandy we could just use `architect deploy` here since it would register + deploy the application at the same time

Reply:
@Zach-at-Arch, I forgot we added that! I could have skipped that in the video, too.